src/components/molecules/DatePickerInput/DatePickerInput.tsx

```diff
@@ -0,0 +1,110 @@
+import { forwardRef } from 'react'
+import { FieldError } from 'react-hook-form'
+import DatePicker, {
+  ReactDatePickerProps,
+  registerLocale,
+} from 'react-datepicker'
+import { Field, Label } from '@radix-ui/react-form'
+import dayjs from 'dayjs'
+import { et } from 'date-fns/locale'
+import InputError from 'components/atoms/InputError/InputError'
+import Icon from 'components/atoms/Icon/Icon'
+import { ReactComponent as Calender } from 'assets/icons/calender.svg'
+import classNames from 'classnames'
+
+import 'react-datepicker/dist/react-datepicker.css'
+import classes from './styles.module.scss'
+
+export interface DatePickerInputProps {
+  name: string
+  className?: string
+  error?: FieldError
+  label?: JSX.Element | string
+  ariaLabel: string
+  message?: string
+  placeholder?: string
+  dateFormat?: string
+  disabled?: boolean
+  value?: string
+  onChange: (value: string) => void
+}
+
+registerLocale('et-EE', et)
+
+const changeDateToString = (dateObject: Date | null | undefined) =>
+  dayjs(dateObject).format('DD/MM/YYYY')
+
+const DatePickerInput = forwardRef<HTMLInputElement, DatePickerInputProps>(
+  function DatePickerInput(props, ref) {
+    const {
+      label,
+      name,
+      disabled,
+      placeholder,
+      dateFormat,
+      className,
+      ariaLabel,
+      message,
+      value,
+      ...rest
+    } = props
+
+    const { onChange } = rest
+
+    const ariaLabelToUse = ariaLabel || (label as string)
+
+    const isWeekday = (date: { getDay: () => number }) => {
+      const day = date.getDay()
+      return day !== 0 && day !== 6
+    }
+
+    const handleDateChange: ReactDatePickerProps['onChange'] = (value) => {
+      return onChange(changeDateToString(value))
+    }
+
+    const splittedValue = value?.split('/')
+    const formattedValue =
+      splittedValue?.[2] + '-' + splittedValue?.[1] + '-' + splittedValue?.[0]
+
+    return (
+      <Field
+        name={name}
+        className={classNames(classes.datePickerContainer, className)}
+      >
+        <Label
+          htmlFor="DatePicker"
+          className={classNames(classes.label, !label && classes.hiddenLabel)}
+        >
+          {label}
+        </Label>
+        <div
+          className={classNames(
+            classes.wrapper,
+            message && classes.errorMessage
+          )}
+        >
+          <DatePicker
```

Comment:
**[blocker]**

Currently when I pick any date in the calendar (for example 11.05.2023), the corresponding day (11.) will be marked on every month in the calendar

<img width="463" alt="Screenshot 2023-05-04 at 08 00 53" src="https://user-images.githubusercontent.com/26221722/236115997-cd184da1-ce00-46a3-87b2-dfff0ff92620.png">